data_generator_tests.py

```diff
@@ -1,28 +1,94 @@
-
+# %% Load Annotations into dataframes
```

Comment:
Thank you for this script! This will help setup my data aug script and make it a lot easier to debug. :)